packages/core/src/http-middlewares/after/throw-for-stale-auth.js

```diff
@@ -1,14 +1,14 @@
 'use strict';
 
-const { stripQueryFromURL } = require('../../tools/http');
-
 const errors = require('../../errors');
 
+/**
+ * The Python-side will raise `RefreshAuthError` when `autoRefresh === true`.
```

Comment:
super minor, but I might refer to the `server` instead of the `Python-side`. it's helps clarify that this happens in the runner, not just in something that uses python.